backend/src/main/java/hu/flowacademy/meteo/dto/StationDto.java

```diff
@@ -16,13 +16,15 @@
     private String name;
     private Double longitude;
     private Double latitude;
+    private int intensity;
 
     public static StationDto toDto(Station station) {
         StationDto stationDto = new StationDto();
         stationDto.setId(station.getId());
         stationDto.setName(station.getName());
         stationDto.setLongitude(station.getLongitude());
         stationDto.setLatitude(station.getLatitude());
+        stationDto.setIntensity((int)Math.floor(Math.random()*98)+1);
```

Comment:
Random.nextInt egyszerubb lenne, van bound paramja is.

Reply:
Javítva